src/AlgoliaSearchModule.cs

```diff
@@ -60,6 +60,8 @@ protected override void OnInit()
 
             DocumentEvents.Delete.Before += HandleDocumentEvent;
             DocumentEvents.Update.Before += HandleDocumentEvent;
+            DocumentEvents.Update.Before += ForceMoveUpdate;
```

Comment:
Both event handlers are already hooked, why not amend existing implementation?

Reply:
Deleted the forcemoveudpate  from delete.before. Update is hooked only once.